### PR TITLE
Remove imagemagick from dockerfile

### LINKED ISSUE #492: remove imagemagick from Dockerfile
Currently, we're installing imagemagick in our Dockerfile. As far as I know, we don't use imagemagick so far.

### PR DESCRIPTION
**Pull request by Joeper214**
## Motivation and Context

Remove unecessary dependency
- Fix #492 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -165,7 +165,7 @@ WORKDIR /home/blueoil
 
 # For development
 RUN apt-get update && apt-get install -y \
-     x11-apps imagemagick
```
Comment (tkng) [inline docker/Dockerfile]: Could you also remove x11-apps?

File: docker/Dockerfile [hunk 2]
```diff
@@ -165,7 +165,7 @@ WORKDIR /home/blueoil
...
+     x11-apps
 
 RUN  apt-get clean && \
```
Comment (tkng) [inline docker/Dockerfile]: Could you remove this line also?

File: lmnet/dockerfiles/cpu.Dockerfile [hunk 3]
```diff
@@ -68,7 +68,6 @@ RUN cd third_party/coco/PythonAPI && pip install -e .
 # For development 
 RUN apt-get update && apt-get install -y \
     x11-apps \
```
Comment (tkng) [inline lmnet/dockerfiles/cpu.Dockerfile]: Could you also remove x11-apps?

File: lmnet/dockerfiles/gpu-volta.Dockerfile [hunk 4]
```diff
@@ -86,7 +86,6 @@ RUN cd third_party/coco/PythonAPI && pip install -e .
 # For development 
 RUN apt-get update && apt-get install -y \
     x11-apps \
```
Comment (tkng) [inline lmnet/dockerfiles/gpu-volta.Dockerfile]: Could you also remove x11-apps?

File: lmnet/dockerfiles/gpu.Dockerfile [hunk 5]
```diff
@@ -68,7 +68,6 @@ RUN cd third_party/coco/PythonAPI && pip install -e .
 # For development 
 RUN apt-get update && apt-get install -y \
     x11-apps \
```
Comment (tkng) [inline lmnet/dockerfiles/gpu.Dockerfile]: Could you also remove x11-apps?

File: lmnet/dockerfiles/multi-gpu.Dockerfile [hunk 6]
```diff
@@ -113,7 +113,6 @@ RUN cd third_party/coco/PythonAPI && pip install -e .
 # For development 
 RUN apt-get update && apt-get install -y \
     x11-apps \
```
Comment (tkng) [inline lmnet/dockerfiles/multi-gpu.Dockerfile]: Could you also remove x11-apps?

File: docker/Dockerfile (hunk 2 above)
Comment (Joeper214) [reply docker/Dockerfile]: You Mean the from line 166-168 right?

Comment (tkng) [reply docker/Dockerfile]: I meant line 166-171 in the original file.

**Comment by Joeper214** [comment]
@tkng Updated thanks!

**Comment by tkng** [comment]
LGTM.

**Pull request merged**